analyzers/src/SonarAnalyzer.Common/Protobuf/AnalyzerReport.proto

```diff
@@ -107,3 +107,17 @@ message FileMetadataInfo {
   bool is_generated = 2;
   string encoding = 3;
 }
+
+// Logging
+enum LogSeverity {
+  UNKNOWN_SEVERITY = 0;
+  DEBUG = 1;
+  INFO = 2;
+  WARNING = 3;
```

Comment:
According to the MMF the severity should be INFO, ERROR or DEBUG.

Reply:
Good catch, MMF is fixed now :D We don't want to fail the analysis (yet), so ERROR shouldn't be there.